### PR TITLE
Fix breaking changes from Lisk SDK 6.0.0-alpha.2 - Closes #654

### LINKED ISSUE #654: Fix breaking changes from Lisk SDK 6.0.0-alpha.2
### Expected behavior

Update breaking changes introduced in Lisk SDK latest alpha to get Lisk Core working

### Actual behavior

Updating Lisk Core with latest alpha fails has SDK introduces several breaking changes

### Steps to reproduce

- Update SDK to latest alpha version

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?

This PR resolves #654 

### How was it solved?

- [x] Update Lisk SDK to v6.0.0-alpha.2
- [x] Resolve all breaking changes

### How was it tested?

Run the following steps:
- `nvm use` / `nvm install`
- `npm run build`
- `./bin/run start --network devnet --api-ipc --console-log info`

### CODE REVIEW

**Review by ManuGowda — COMMENTED** [review]
@sameersubudhi i think you have to comment the piece of code that @nagdahimanshu mentioned? otherwise core wont run?

**Comment by sameersubudhi** [comment]
> @sameersubudhi i think you have to comment the piece of code that @nagdahimanshu mentioned? otherwise core wont run?

@ManuGowda As discussed, will comment out the piece of code locally when setting up the node.

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

File: src/application/modules/legacy/commands/reclaim.ts [hunk 1]
```diff
@@ -40,8 +40,12 @@ import {
 } from '../schemas';
 import { ReclaimParamsData, LegacyStoreData, TokenIDReclaim } from '../types';
 
-const { LiskValidationError, validator } = liskValidator;
-const { getLegacyAddressFromPublicKey, getAddressFromPublicKey } = cryptography;
+// eslint-disable-next-line prefer-destructuring
+const validator: liskValidator.LiskValidator = liskValidator.validator;
```
Comment (ManuGowda) [inline src/application/modules/legacy/commands/reclaim.ts:44]: lets avoid this and use directly `liskValidator.validator.validate`?

File: src/application/modules/legacy/commands/register_keys.ts [hunk 2]
```diff
@@ -33,8 +33,12 @@ import {
...
 import { registerKeysData } from '../types';
 
-const { getAddressFromPublicKey } = cryptography;
-const { LiskValidationError, validator } = liskValidator;
+const {
+	address: { getAddressFromPublicKey },
+} = cryptography;
+
+// eslint-disable-next-line prefer-destructuring
+const validator: liskValidator.LiskValidator = liskValidator.validator;
```
Comment (ManuGowda) [inline src/application/modules/legacy/commands/register_keys.ts:41]: same here and in other places too

File: package.json [hunk 3]
```diff
@@ -101,20 +101,20 @@
 		}
 	},
 	"dependencies": {
-		"@liskhq/lisk-framework-faucet-plugin": "0.2.0-alpha.31866",
-		"@liskhq/lisk-framework-forger-plugin": "0.3.0-alpha.31866",
-		"@liskhq/lisk-framework-monitor-plugin": "0.3.0-alpha.31866",
-		"@liskhq/lisk-framework-report-misbehavior-plugin": "0.3.0-alpha.31866",
+		"@liskhq/lisk-framework-faucet-plugin": "0.2.0-alpha.2",
+		"@liskhq/lisk-framework-forger-plugin": "0.3.0-alpha.2",
+		"@liskhq/lisk-framework-report-misbehavior-plugin": "0.3.0-alpha.2",
```
Comment (ManuGowda) [inline package.json:106]: I think the old order was correct alphabetically, same with sdk and commander below, commander first then sdk

File: src/application/modules/legacy/commands/reclaim.ts (hunk 1 above)
Comment (sameersubudhi) [reply src/application/modules/legacy/commands/reclaim.ts:44]: Refactoring it to any other form results in the following error:
```
Assertions require every name in the call target to be declared with an explicit type annotation.ts (2775)
```

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**